components/Navbar/NavbarMobile/HamburgerMenu.tsx

```diff
@@ -0,0 +1,37 @@
+import clsx from 'clsx'
+import React, { useState } from 'react'
+
+export default function HamburgerMenu(props) {
+  const className =
+    'h-0.5 w-5 bg-white transition-all ease-in-out origin-top-right'
+
+  const aria_label = props.isOpen ? 'Close Menu' : 'Open Menu'
+
+  return (
+    <div
+      className="mr flex h-full w-10 cursor-pointer flex-col items-start justify-center overflow-hidden border-r border-white/25 p-1"
+      onClick={props.clicked}
+      role="button"
+      aria-label={aria_label}
+      aria-expanded={props.isOpen}
+      aria-controls="lesson-navigation-menu"
+    >
+      <div
+        className={clsx(className, 'mb-[5px]', {
+          'transform-none': !props.isOpen,
+          'translate-x-0 -translate-y-[1.8px] -rotate-45': props.isOpen,
+        })}
+        aria-hidden="true"
+      />
+
+      <div
+        className={clsx(className, 'mt-[2px]', {
+          'w-[15px] transform-none': !props.isOpen,
+          'w-[20px] translate-x-[1.2px] translate-y-[3px] rotate-45':
```

Comment:
Do we really need subpixel values like `1.2px`? I'm not a big fan of arbitrary values in tailwind, I think they become hard to maintain, let alone subpixel ones.

Reply:
I experimented with changing the values to the nearest integer pixel here, but it made the "cross state" of the hamburger menu slightly off when displayed.

On a smaller screen, it would be more evident, so I didn't touch the subpixel value here for now. But I would love to hear your take on this point.

Reply:
This is a unique situation where we do need super fine-grained control to get the bars to cross visually in just the right way, so I think it's warranted here. Of course not something you want to do all over the place.

Reply:
I agree with @GBKS! Also, as we discussed on the call last Tuesday, to correctly render specific cases like hamburger cross, we can work with arbitrary subpixel values.